src/account.rs

```diff
@@ -550,11 +609,18 @@ impl Account {
         order_parameters.insert("type".into(), order.order_type);
         order_parameters.insert("quantity".into(), order.qty.to_string());
 
+        if let Some(stop_price) = order.stop_price {
+            let valid_stop_price = format!("{:.7}", stop_price);
```

Comment:
The rounding required is symbol-dependent - some need rounding to 7 digits, others to 3 or to 6. This rounding is not really consistent - other functions (e.g. other order types) also don't do this. I think it's the responsibility of the user of this API to provide correctly rounded numbers.

Reply:
Thank you very much! It would be nice to add a better error handling in this case? I will sort this as soon as possible!

Reply:
I don't think it will help much, the problem relies in the core of the library because the type of value `price` in most of the functions is a float and it doesn't allow rounding. We need to either transform value into a String or add a option to round it if required.

Reply:
This can help: https://stackoverflow.com/a/63214916

Reply:
I'm doing this kind of rounding in my own application (with amount of digits based on the significance of that particular symbol pair) and then I send the rounded number to a binance-rs function, and never had any issues.